cmd/maestro/common/otlp_sdk.go

```diff
@@ -0,0 +1,99 @@
+package common
```

Comment:
Maybe we should place the initialization to a place where it can be reused by the frontend too. Wdyt?

E.g. here:  https://github.com/Azure/ARO-HCP/tree/main/internal

Reply:
I agreed but should it be somewhere else instead of "internal" package? Internal pkg usually means it is not exportable or should not be exportable.

Reply:
github.com/openshift-online/ocm-common might be the best option since it's already imported by https://gitlab.cee.redhat.com/service/uhc-clusters-service/ too. And it should be ok to import it in https://github.com/Azure/ARO-HCP/ too.